CHANGELOG.md

```diff
@@ -1,20 +1,31 @@
 # Change Log
 
+## v2.0.0
+
+-   `ChannelValue`'s value attribute now accepts both `string` type and `number` type.
+    _Breaking Changes_
+-   `pxb-scorecard` is now `pxb-score-card`.
```

Comment:
Scorecard and info list item didn't exist in 1.3.0, can you update your breaking changes section?

Reply:
Spacer is new too, could you add it as well?